packages/now-cli/src/commands/secrets.js

```diff
@@ -141,11 +152,11 @@ async function run({ output, token, contextName, currentTeam, ctx }) {
       return exit(1);
     }
 
-    const list = await secrets.ls();
+    const { secrets: list, pagination } = await secrets.ls(nextTimestamp);
     const elapsed = ms(new Date() - start);
```

Comment:
I realize the code was already like this, but might as well avoid a type cast:

```suggestion
    const elapsed = ms(Date.now() - start);
```